example/go.mod

```diff
@@ -1,26 +1,103 @@
 module github.com/google/go-github/v66/example
 
-go 1.21
+go 1.22.5
+
+toolchain go1.23.2
 
 require (
 	github.com/ProtonMail/go-crypto v0.0.0-20230828082145-3c4c8a2d2371
 	github.com/bradleyfalzon/ghinstallation/v2 v2.0.4
 	github.com/gofri/go-github-ratelimit v1.0.3
 	github.com/google/go-github/v66 v66.0.0
-	golang.org/x/crypto v0.21.0
-	golang.org/x/term v0.18.0
-	google.golang.org/appengine v1.6.7
+	github.com/sigstore/sigstore-go v0.6.2
```

Comment:
Why is `github.com/sigstore/sigstore-go` pulling in an enormous boat load of dependencies below with it?

Reply:
The library does a lot, including signing, hashing, provisioning of short-lived certs etc. If you'd prefer not to pull in so many indirect deps in the example module I can break it out into its own go module or host the example code in a different repo.